llama_stack/distribution/routers/routing_tables.py

```diff
@@ -608,8 +608,8 @@ async def unregister_toolgroup(self, toolgroup_id: str) -> None:
         tool_group = await self.get_tool_group(toolgroup_id)
         if tool_group is None:
             raise ValueError(f"Tool group {toolgroup_id} not found")
-        tools = (await self.list_tools(toolgroup_id)).data
-        for tool in tools:
+        tools = await self.list_tools(toolgroup_id)
+        for tool in tools.data:
```

Comment:
Would be good to handle case where data is not an attribute

Reply:
@franciscojavierarceo - good point. Done.

Reply:
Thank you!